css-fonts-5/Overview.bs

```diff
@@ -59,7 +59,63 @@ rules defined in CSS Fonts Level 4.
 This specification is currently a delta to the CSS Fonts Level 4 specification.
 Do not assume that if something is not here, it has been dropped.
 
+<h2 id="text-scale-meta">
+Text-Scale <code class=html>&lt;meta&gt;</code> element</h2>
 
+A document with a <code>&lt;meta></code> tag whose <code>name</code> attribute
+is a <a>ASCII case-insensitive</a> match for
+<dfn lt=text-scale><code>"text-scale"</code></dfn> is recognized as setting the
+initial font size of the document. The value of the <code>content</code>
+attribute must be an <a>ASCII case-insensitive</a> match for one of the
+recognized keywords.
+
+Documents without this <code>&lt;meta></code> tag will have an assumed default
+value of <code>legacy</code>.
+
+<h3 id="text-scale-meta-keywords">
+Keywords</h3>
+
+The recognized keywords in the [=text-scale=]
+<code class=html>&lt;meta&gt;</code> element are:
+
+<ul>
+	<li><code class="index" lt="legacy!!text-scale-meta">legacy</code></li>
+	<li><code class="index" lt="scale!!text-scale-meta">scale</code></li>
+</ul>
+
+<h3 id="legacy-keyword">The 'legacy' keyword</h3>
+
+The <dfn for="text-scale" export><code>legacy</code></dfn> property is
+recognized in the [=text-scale=] content attribute value.
+
+When the value of the [=text-scale=] content attribute is
+<a for="text-scale">legacy</a> the user agent should set the initial font size
+to 16px multiplied by the font scale factor the user has chosen from any settings <i>provided by the user agent</i>. The ''preferred-text-scale'' value must be 1 on desktop platforms. On mobile:
+<ul>
+	<li>if the operating system provides a text scale setting AND the UA hasn't already applied that factor to the initial font size, ''env()/preferred-text-scale'' returns the mulitplier that the user has chosen in the operating system's text scale setting.
+
+	Note: As of this writing, all combinations of Android, iOS, Gecko, WebKit, Blink satisfy this first condition.
+
+	</li>
+	<li>Otherwise ''env()/preferred-text-scale'' returns 1.</li>
+</ul>
+
+<h3 id="scale-keyword">The 'scale' keyword</h3>
+
+The <dfn for="text-scale" export><code>scale</code></dfn> property is
+recognized in the [=text-scale=] content attribute value.
+
+When the value of the [=text-scale=] content attribute is
+<a for=text-scale>scale</a> the user agent may determine the initial font size
+based on a combination of the operating system's text scale setting and the user agent's text scale setting. The
```

Comment:
"may" or "must"? Shouldn't this be a "must"? Can we define more definitively what this is a "must" about? Like,

The `medium` font size must match the user's preferred paragraph text size as determined from a combination of OS and UA preferences.

Reply:
Great, taken verbatim.